float_fmt arg for gmsh writer is missing

I'm using last tag 4.0.11.

The float_fmt arg is implemented in the _gmsh\*\*.py helpers but the gmsh main.py is not passing this argument.
gmsh main.py write signature :
`def write(filename, mesh, fmt_version="4.1", binary=True):`

This causes a crash when using the meshio.writer with float_fmt as argument.

``` bash
kwargs = {'binary': False, 'float_fmt': '.15f'}

>           "gmsh22": lambda f, m, **kwargs: write(f, m, "2.2", **kwargs),
            "gmsh": lambda f, m, **kwargs: write(f, m, "4.1", **kwargs),
        },
    )
E   TypeError: write() got an unexpected keyword argument 'float_fmt'
```

--- a/Makefile
+++ b/Makefile
@@ -15,8 +15,9 @@ upload: clean setup.py
 	# Make sure we're on the master branch
 	@if [ "$(shell git rev-parse --abbrev-ref HEAD)" != "master" ]; then exit 1; fi
 	rm -f dist/*
-	python3 setup.py sdist
-	python3 setup.py bdist_wheel
+	# python3 setup.py sdist bdist_wheel
+	# https://stackoverflow.com/a/58756491/353337
+	python3 -m pep517.build --source --binary .
 	twine upload dist/*
 
 publish: tag upload

--- a/logo/logo.py
+++ b/logo/logo.py
@@ -70,9 +70,11 @@ def create_logo2(y=0.0):
 
     mesh = pygmsh.generate_mesh(geom)
     # return mesh.points, mesh.cells["triangle"]
+    X = mesh.points
+    cells = mesh.get_cells_type("triangle")
 
     X, cells = optimesh.cvt.quasi_newton_uniform_full(
-        mesh.points, mesh.cells["triangle"], 1.0e-10, 100
+        X, cells, 1.0e-10, 100, verbose=True
     )
     return X, cells
 
@@ -80,7 +82,8 @@ def create_logo2(y=0.0):
 if __name__ == "__main__":
     X, cells = create_logo2(y=0.08)
 
-    meshio.write_points_cells("logo.svg", X, {"triangle": cells})
+    mesh = meshio.Mesh(X, {"triangle": cells})
+    meshio.svg.write("logo.svg", mesh, force_width=300)
 
     X = numpy.column_stack([X[:, 0], X[:, 1], numpy.zeros(X.shape[0])])
-    meshio.write_points_cells("logo.vtk", X, {"triangle": cells})
+    meshio.write("logo.vtk", meshio.Mesh(X, {"triangle": cells}))

--- a/meshio/gmsh/_gmsh41.py
+++ b/meshio/gmsh/_gmsh41.py
@@ -273,11 +273,11 @@ def _read_periodic(f, is_ascii, data_size):
     return periodic
 
 
-def write(filename, mesh, binary=True):
-    write4_1(filename, mesh, binary=binary)
+def write(filename, mesh, float_fmt=".15e", binary=True):
+    write4_1(filename, mesh, float_fmt=float_fmt, binary=binary)
 
 
-def write4_1(filename, mesh, binary=True):
+def write4_1(filename, mesh, float_fmt=".15e", binary=True):
     """Writes msh files, cf.
     <http://gmsh.info/doc/texinfo/gmsh.html#MSH-file-format>.
     """
@@ -314,7 +314,7 @@ def write4_1(filename, mesh, binary=True):
             _write_physical_names(fh, mesh.field_data)
 
         _write_entities(fh, cells, binary)
-        _write_nodes(fh, mesh.points, mesh.cells, binary)
+        _write_nodes(fh, mesh.points, mesh.cells, binary, float_fmt)
         _write_elements(fh, cells, binary)
         if mesh.gmsh_periodic is not None:
             _write_periodic(fh, mesh.gmsh_periodic, binary)
@@ -359,7 +359,7 @@ def _write_entities(fh, cells, binary):
     return
 
 
-def _write_nodes(fh, points, cells, binary):
+def _write_nodes(fh, points, cells, binary, float_fmt):
     fh.write(b"$Nodes\n")
 
     # The entity_dim and entity_tag in the $Elements section must correspond to an
@@ -414,7 +414,7 @@ def _write_nodes(fh, points, cells, binary):
         )
         numpy.arange(1, 1 + n, dtype=c_size_t).tofile(fh, "\n", "%d")
         fh.write(b"\n")
-        numpy.savetxt(fh, points, delimiter=" ")
+        numpy.savetxt(fh, points, delimiter=" ", fmt="%" + float_fmt)
 
     fh.write(b"$EndNodes\n")
     return

--- a/meshio/gmsh/main.py
+++ b/meshio/gmsh/main.py
@@ -87,7 +87,7 @@ def _read_header(f):
     return fmt_version, data_size, is_ascii
 
 
-def write(filename, mesh, fmt_version="4.1", binary=True):
+def write(filename, mesh, fmt_version="4.1", binary=True, float_fmt=".15e"):
     """Writes a Gmsh msh file.
     """
     try:
@@ -102,7 +102,7 @@ def write(filename, mesh, fmt_version="4.1", binary=True):
                 )
             )
 
-    writer.write(filename, mesh, binary=binary)
+    writer.write(filename, mesh, binary=binary, float_fmt=".15e")
 
 
 register(

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,6 +1,6 @@
 [metadata]
 name = meshio
-version = 4.0.11
+version = 4.0.12
 author = Nico Schlömer et al.
 email = nico.schloemer@gmail.com
 description = I/O for many mesh formats

--- a/setup.py
+++ b/setup.py
@@ -1,4 +1,0 @@
-from setuptools import setup
-
-if __name__ == "__main__":
-    setup()

